packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -205,6 +206,11 @@ export abstract class QueryEditorStore {
   initState = ActionState.create();
   queryBuilderState: QueryBuilderState;
   exportState?: QueryExportState | undefined;
+  isQueryLoaderOpen = false;
```

Comment:
I would move this out to its own state.

Reply:
+1, we have enough state here to justify that we need its own state

Reply:
@YannanGao-gs let's not forget about this :)